scripts/populate_mock_db.py

```
@@ -0,0 +1,95 @@
+    for i in range(50):
+        name = f"Crew Member {i:02d}"
+        role = roles[i % len(roles)]
+        skills = "BA TTR LGV" if i % 4 == 0 else "BA TTR"
+        c.execute(
+            "INSERT INTO crew (name, role, skills, contract_hours) VALUES (?, ?, ?, ?)",
+            (name, role, skills, "42 hours"),
+        )
+
+        crew_id = c.lastrowid
+        # Available now until tomorrow
+        now = datetime.now()
```

Ruff (0.15.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)